src/App.jsx

```diff
@@ -1,28 +1,29 @@
-import logo from "./logo.svg";
-import "./App.css";
-import { BrowserRouter, Route, Routes, route } from "react-router-dom";
-import { Provider } from "react-redux";
-import store from "./redux/mainStore";
-import { useDispatch } from "react-redux";
-import { useEffect } from "react";
-import { fetchData } from "./redux/data/dataSlice";
+// import logo from './logo.svg';
+import './App.css';
 
-import Navbar from "./components/Navbar";
-import Home from "./components/Home";
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import { fetchData } from './redux/data/dataSlice';
+
+import Navbar from './components/Navbar';
```

Comment:
- [ ] Remember this is a mobile page design and I can see your mobile version has two different designs.ie the single column and the double column. Please according to the original design and guidelines making it clear we follow it, there are two columns with a lighter and thick color mixed in a check form. 
- your two column design ok, but the light and thick color are on their separate columns instead of interchanging with each other like in the original design. See example below.
Your designs
![Screenshot (1193)_LI](https://github.com/benja27/forex-currency-exchange-rate/assets/65298048/9f51fe6f-2182-4029-bcbb-696741bfdb68)
![Screenshot (1194)_LI](https://github.com/benja27/forex-currency-exchange-rate/assets/65298048/8f27c666-1ae4-4240-be36-3b8db42485f3)
Expected and original design
![Screenshot (1195)_LI](https://github.com/benja27/forex-currency-exchange-rate/assets/65298048/849ed63e-6dc6-41f8-a9d7-e14600f20dbf)

Reply:
done